data/roles.js

```diff
@@ -388,6 +388,13 @@ const roleData = {
                 "That person will be resurrected.",
             ],
         },
+        "Santa": {
+            alignment: "Village",
+            description: [
+                "Gives out various items at night.",
+                "Can visit someone at night to know their alignment if not visited.",
```

Comment:
the second sentence sounds awkward to me

Reply:
What would you change it to?

Reply:
Gives a present to one player every night.
If not visited, can also learn if another player is naughty or nice.